### PR TITLE
Add media queries to make the mobile version responsive on desktop sc…

### PR DESCRIPTION
**Pull request by KingKowa**
Added media queries to make sure the website displays correctly on desktop screens.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,18 +10,34 @@
...
-                <h1>I’m Mariane Glad to see you!</h1>
+                <h1>I’m Mariane Glad to see you! 
+
+                </h1>
             </div>
-            <div>
+            <div >
                 <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 
             </div>
```
Comment (ShahierNashaat) [inline index.html:43]: - [ ] Please remove the bar at the top of the header and add a background image to the header.

Your design                       |Figma design               
:--------------------------------: | :-----------------------------------:
Screenshot from 2022-04-29 20-32-30|Screenshot from 2022-04-29 20-35-00

File: styles.css [hunk 2]
```diff
@@ -450,3 +461,427 @@ h3 {
...
+    row-gap: 142px;
+    width: 1440px;
+    height: 2694px;
+    padding-left: 142px;
+    padding-right: 142px;
+    padding-top: 142px;
+    padding-bottom: 142px;
+    margin: 0 auto;
+    background-color: #e5e5e5;
+  }
```
Comment (ShahierNashaat) [inline styles.css:578]: - [ ] Kindly be noticed that there is a horizontal scroll in the desktop size of your portfolio so suggested changing the width in some divs from 1440px to 100% also remove some padding that causes this horizontal scroll.

Screenshot from 2022-04-29 20-33-15

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @KingKowa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @KingKowa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] [OPTIONAL] On the mobile section there's an additional width you can delete it, to make your design perfect.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**